### PR TITLE
Additional WAF refinements to reduce false positives

### PR DESCRIPTION
**Pull request by rin-skylight**
# DEVOPS PULL REQUEST

## Related Issue

- This should hopefully be the last stage of false positive identification and removal. We've gone from several thousand false positives per day to only a few hundred catches, about 95-99% of which are valid.

## Changes Proposed

- Add exceptions for analytics cookies
- Add exceptions for CSV Uploader variable arguments
- Add additional SQL injection rule exception

## Additional Information

- This will be allowed to settle for 24-48 hours. If no other problematic requests are found, one more PR will be issued to activate "Prevention" mode.

### CODE REVIEW

**Review by emmastephenson — APPROVED** [review]
LGTM, thanks for all your hard work in chasing down these exceptions Rin!

File: ops/services/web_application_firewall/main.tf [hunk 1]
```diff
@@ -129,6 +144,7 @@ resource "azurerm_web_application_firewall_policy" "sr_waf_policy" {
           "942200",
           "942260",
           "942330",
+          "942361",
```
Comment (emyl3) [inline ops/services/web_application_firewall/main.tf:147]: I'm seeing [here](https://docs.microsoft.com/en-us/azure/web-application-firewall/ag/application-gateway-crs-rulegroups-rules?tabs=owasp32) that `942361` is excluding `Detects basic SQL injection based on keyword alter or union`.

For my own curiosity, is this a commonly excluded rule or something specific to our application that we need to do this?

**Review by emyl3 — APPROVED** [review]
From what I can understand, it looks good to me! I had one question though 😹

File: ops/services/web_application_firewall/main.tf (hunk 1 above)
Comment (rin-skylight) [reply ops/services/web_application_firewall/main.tf:147]: Great question! This is specific to our application. In certain areas, such as a street or facility name, the word "Union" is perfectly valid. As an example, here in Chicago, we have a street called Union Avenue; if this rule were left in place, any testing facility or patient who resides on that street wouldn't be able to use the application, as the WAF would prevent requests with their information from going through.

This is one of the downsides with a WAF...default implementations tend to be pretty strict, and we can run into trouble trying to loosen things enough to run the application. That's why this process can routinely take a few weeks.

Comment (emyl3) [reply ops/services/web_application_firewall/main.tf:147]: Oh wow! I see thank you. That's a very helpful example and explanation!! 😹

**Review by alismx — COMMENTED** [review]
Just realized this was already merged. :face_with_spiral_eyes: Have a completely unnecessary post-merge approval.

**Pull request merged**